rviz_ogre_vendor/package.xml

```diff
@@ -30,6 +30,7 @@
   <depend>libxaw</depend>
   <depend>libxrandr</depend>
   <depend>opengl</depend>
+  <depend>pugixml-dev</depend>
```

Comment:
Adding this dependency is unfortunate, because we'll need to package it for Windows.

But I'm actually wondering if we can avoid it completely.  It looks like only 2 things in OGRE use it; the XMLConverter and the DotSceneLoader.  I don't think either of those is required for RViz, so we could get away with changing the CMakeLists.txt to have:

```
list(APPEND extra_cmake_args "-DOGRE_BUILD_TOOLS:BOOL=OFF")
list(APPEND extra_cmake_args "-DOGRE_BUILD_PLUGIN_DOT_SCENE:BOOL=OFF")
```

Reply:
Changes have been made.